package.json

```diff
@@ -2,7 +2,7 @@
   "name": "fastboot",
   "version": "1.0.0-rc.0",
   "description": "Library for rendering Ember apps in node.js",
-  "main": "src/index.js",
+  "main": "dist/cjs/index.js",
```

Comment:
I believe it is getting changed by https://github.com/monegraph/broccoli-module-alchemist, when it detects that you are running a version of node that isn't compatible with `src/` (I _think_ that Node 4.x and above would not have this issue).